src/templates/atoms.njk

```diff
@@ -2877,6 +2766,199 @@
             </div>
         </div>
 
+        <h4 class="u-margin-top-xl">Sortable tables</h4>
+        <div class="d">
+            <div>
+                <table class="a-table">
+                    <caption>Sortable table with one unsorted column</caption>
+                    <thead>
+                        <tr>
+                            <th>Snack</th>
+                            <th class="is-collapsed" aria-sort="none">
+                                <button class="a-table__sort-button a-table__sort-button--unsorted">Description</button>
+                            </th>
+                            <th class="u-text-right">Stock</th>
+                            <th class="u-text-right">Price</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td class="u-text-bold">Ice cream</td>
+                            <td class="small">Tasty, frozen treat for a hot day</td>
```

Comment:
Vraagje: moeten we deze niet aanpassen naar `.u-text-small`?

Reply:
Mondeling besproken. 😉